packages/turf-ellipse/index.js

```diff
@@ -0,0 +1,68 @@
+const polygon = require('@turf/helpers').polygon;
+const lengthToDegrees = require('@turf/helpers').lengthToDegrees;
+const getCoord = require('@turf/invariant').getCoord;
+
+/**
+ * Takes a {@link Point} and calculates the ellipse polygon given two axes in degrees and steps for precision.
+ *
+ * @param {Feature<Point>|Array<number>} center center point
+ * @param {number} xRadius radius of the ellipse along the x-axis
+ * @param {number} yRadius radius of the ellipse along the y-axis
+ * @param {Object} [options={}] Optional parameters
+ * @param {number} [options.steps=64] number of steps
+ * @param {string} [options.units='kilometers'] unit of measurement for axes
+ * @param {Object} [options.properties={}] properties
+ * @returns {Feature<Polygon>} ellipse polygon
+ * @example
+ * const center = [-75.9975, 40.730833];
+ * const xRadius = 0.5;
+ * const yRadius = 0.1;
+ * const options = {steps: 10, properties: {foo: 'bar'}};
+ * const ellipse = turf.ellipse(center, xRadius, yRadius, options);
+ *
+ * //addToMap
+ * const addToMap = [turf.point(center), ellipse]
+ */
+module.exports = function (center, xRadius, yRadius, options) {
```

Comment:
We are using ES Modules now + this type of module should also be using `export default`.

```js
function ellipse(center, xRadius, yRadius, options) {
   ...
}
export default ellipse
```